thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/controller.js

```diff
@@ -336,7 +352,7 @@ export default Ember.Controller.extend({
     this.set('isEditAlertSuccess', true);
     Ember.run.later((function() {
       that.clearAll();
-      this.transitionToRoute('manage.alerts');
+      that.transitionToRoute('manage.alerts');
```

Comment:
### Blocking comment: 
let's refrain from using `that` . The `Ember.run.later` api allows to pass the context before the callback. 
https://www.emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/bool?anchor=notEmpty

```
Ember.run.later(this, (function() {
})
```

Reply:
Good to know!